target/linux/ar71xx/files/arch/mips/ath79/mach-cf-e316n-v2.c

```diff
@@ -355,6 +408,43 @@ static void __init cf_e320n_v2_setup(void)
 MIPS_MACHINE(ATH79_MACH_CF_E320N_V2, "CF-E320N-V2", "COMFAST CF-E320N v2",
 	     cf_e320n_v2_setup);
 
+static void __init cf_e355ac_common_setup(void)
```

Comment:
Please, drop this function.
Other `*_common_setup()` functions are called by more than one board init functions.